gtest/src/manager.rs

```diff
@@ -445,6 +449,61 @@ impl ExtManager {
 
         core_processor::handle_journal(journal, self);
     }
+
+    #[allow(dead_code)]
+    pub(crate) fn execute_custom_function(
+        &mut self,
+        source: ProgramId,
+        program_id: &ProgramId,
+        message_option: Option<IncomingMessage>,
+        function_name: &str,
+    ) -> Vec<u8> {
+        let mut executor = self.get_executor(source, program_id, message_option);
+        let (result, journal) = executor.call_function(function_name);
+        core_processor::handle_journal(journal, self);
+        result
+    }
+
+    #[allow(dead_code)]
+    pub(crate) fn execute_custom_void_function(
+        &mut self,
+        source: ProgramId,
+        program_id: &ProgramId,
+        message_option: Option<IncomingMessage>,
```

Comment:
No need to highlight that this is an option.
```suggestion
        message: Option<IncomingMessage>,
```